src/js/fluxcontroller.js

```diff
@@ -489,6 +493,10 @@ define(function (require, exports, module) {
                                 transferQueue.active(), transferQueue.pending());
                         }
 
+                        if (action.action.noResetOnFailure) {
+                            nextAction.action.noResetOnFailure = action.action.noResetOnFailure;
```

Comment:
I also think that it probably _should_ be the case that `allowFailure` is transitive across the transfer relation. Transfers should be transparent to the caller. I don't know why we would want to allow a `batchPlay` to fail but not a transfer. Often we clean actions up by modularizing them into transferrable actions, and I don't think that should change the semantics of the action.